src/components/Button.scss

```diff
@@ -50,8 +50,6 @@ $Button-padding: 0 scaleGrid(2);
 
   &.is-active {
     background-color: $Button-bg-hover;
-    border-bottom-width: 1px;
-    border-top-width: $Button-border-bottom-width;
   }
```

Comment:
@williamdingwall What do you think about this change? Adding the heavy border to the top on active state always seemed a little weird to me... especially in the button group, but maybe that's just me!